src/test/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsAutoListMappingTest.java

```diff
@@ -0,0 +1,288 @@
+    @Test
+    void testHasOrgOtherPartiesNone() {
+        List<CcdValue<OtherParty>> otherParties = List.of(
+                new CcdValue<OtherParty>(OtherParty.builder()
+                        .rep(Representative.builder()
+                                .hasRepresentative("Yes")
+                                .organisation("")
+                                .build())
+                        .build()),
+                new CcdValue<OtherParty>(OtherParty.builder()
+                        .rep(Representative.builder()
+                                .hasRepresentative("Yes")
+                                .build())
+                        .build()));
+
+        boolean result = HearingsAutoListMapping.hasOrgOtherParties(otherParties);
+
+        assertThat(result).isFalse();
+    }
+
+    @DisplayName("When no other party has a rep with a non blank name, hasOrgOtherParties should return false")
+    @Test
+    void testHasOrgOtherPartiesNull() {
+        boolean result = HearingsAutoListMapping.hasOrgOtherParties(null);
+
+        assertThat(result).isFalse();
+    }
+
+    @DisplayName("When hasRepresentative is Yes and organisation not blank isRepresentativeOrg should return True")
+    @Test
+    void testIsRepresentativeOrg() {
+        Representative rep = Representative.builder()
+                .hasRepresentative("Yes")
+                .organisation("Test")
+                .build();
+
+        boolean result = HearingsAutoListMapping.isRepresentativeOrg(rep);
+
+        assertThat(result).isTrue();
+    }
+
+    @DisplayName("When hasRepresentative is No, blank or null, isRepresentativeOrg should return False")
+    @ParameterizedTest
+    @ValueSource(strings = {"No"})
+    @NullAndEmptySource
+    void testIsRepresentativeOrgNull(String value) {
+        Representative rep = Representative.builder()
+                .hasRepresentative(value)
+                .organisation("Test")
+                .build();
+
+        boolean result = HearingsAutoListMapping.isRepresentativeOrg(rep);
+
+        assertThat(result).isFalse();
+    }
+
+    @DisplayName("When Organisation is null, isRepresentativeOrg should return False")
+    @ParameterizedTest
+    @NullAndEmptySource
+    void testIsRepresentativeOrgBlankOrg(String value) {
+        Representative rep = Representative.builder()
+                .hasRepresentative("Yes")
+                .organisation(value)
+                .build();
+
+        boolean result = HearingsAutoListMapping.isRepresentativeOrg(rep);
+
+        assertThat(result).isFalse();
+    }
+
+    @DisplayName("When Representative is null, isRepresentativeOrg should return False")
+    @Test
+    void testIsRepresentativeOrgRepNull() {
+        boolean result = HearingsAutoListMapping.isRepresentativeOrg(null);
+
+        assertThat(result).isFalse();
+    }
+
+    @DisplayName("When hearingType is Paper, isPaperCaseAndNoPO return True")
```

Comment:
minor/optional: if you want to change the method name remember to update these descriptions as well